.travis.yml

```diff
@@ -14,7 +14,7 @@ cache:
   - node_modules
 before_install:
   - npm install -g npm@5.5.1
-  - if [ -n "${VERACODE_WRAPPER_VERSION}" ]; then wget https://repo1.maven.org/maven2/com/veracode/vosp/api/wrappers/vosp-api-wrappers-java/$VERACODE_WRAPPER_VERSION/vosp-api-wrappers-java-$VERACODE_WRAPPER_VERSION.jar ; fi
```

Comment:
Hmm, if I understand correctly, with the change to the condition the scan will only ran on tagged commits. What did the `VERACODE_WRAPPER_VERSION` env variable represent? and why did we use it before?

Reply:
To prevent fork from being scanned. Tagged commits will have the same effect, @olamothe 's recommendation